msg_package_template/package.xml

```diff
@@ -33,6 +33,8 @@
   <buildtool_depend>catkin</buildtool_depend>
   <buildtool_depend>catkin_simple</buildtool_depend>
   
-  <depend>roscpp</depend>
+  <depend>message_generation</depend>
```

Comment:
message_generation should be buildtool_depend.
and check if we can remove msg_runtime

Reply:
can't remove it, dependent packages aren't able to find the message headers then.